tyr/clusters/mongo.py

```diff
@@ -2,6 +2,7 @@
 from tyr.servers.mongo import MongoDataNode, MongoArbiterNode
 import time
 import json
+import boto
```

Comment:
Since we're no longer querying Amazon VPC for the subnets associated with a VPC, this import is no longer necessary.